_data/projects/1-TheOpenSourceGuide.yml

```diff
@@ -0,0 +1,23 @@
+---
+name: The Open Source Guide
+desc: 'This repository is designed for helping begineers getting started with Open Source Development'
```

Comment:
@josharsh This looks more of a guide and less of a contribution project. This might be good for Contributors to read and all. But we focus on projects here that are good for users to contribute. I can't see any open issues where users can contribute.